### ISSUE TITLE
project status

### ISSUE THREAD

**Issue by stefanocrosta** [circus-tent/circus#1046]
I got interested into Circus after reading some nice blog posts and the project description and philosophy; but looking at the github repo it looks like development has stopped about a year ago, and even pull requests are not actively integrated.

could you please share what the project status and future plans are please?

Is it worth investing into this for use in production - and of course potentially contributing - or not?

Thanks and best,

PS. I tried sending this message to the mailing list 10 days ago, but the message did not gotten moderation approval yet (nor denial) which is not promising :-(

**Comment by k4nar** [comment]
tl;dr: Circus is not dead, but contributors are welcome :) .

I've been trying to cope with the issues & the PRs for the past few years, but with a limited success. I don't have much time to work on it, and it's a rather complex project, dealing with complicated stuff like system programming, ZeroMQ or asynchronous programming.

The main issue on my side is that I'm not using Circus anymore. Not because I've found a better replacement, but because I don't need it. I'm still interested by the project, but it's hard to make sacrifices for something that you're not using.

Other contributors are still active sometimes. For example the main author, @tarekziade, merged some PRs lately.

That being said, we would be happy to coach other people interested in contributing to Circus on a regular basis! As you said, the mailing list is pretty dead, but we could use Github or even our good old IRC channel (#mozilla-circus on Freenode).

Regarding Circus in production, I think a lot of people still use it successfully. It's still downloaded and issues/PRs are opened regularly.
I don't think that a new & better replacement exists today, although there are other very good process managers (like Supervisor for instance). Maybe systemd could supersede them, but I'm not sure it's really ready for this kind of usage today.

**Comment by stefanocrosta** [comment]
Merci Yannick!

that's good news, I'll take a serious look then! 
FYI the thing i'm most interested in for Circus is the API usage - dynamically defining process to run rather than through setting files; as you say, the closest alternative is supervisor, and for my usage in particular with  https://github.com/mnaberez/supervisor_twiddler
Re. systemd, it is extremely powerful but also difficult to master... and does not seem to fit my current specific need.

I think the Circus approach is very cool so hopefully it will keep flying!

**Comment by idchlife** [comment]
@k4nar why you found circus is not needed for your tasks? You started using docker or just jumped to different tasks which do not need process management?

Also, I think this issue can be reopened, for newcomers like myself to learn about project status peacefully and not via personal experience etc

Thanks!
